cargo-pgx/src/command/test.rs

```diff
@@ -35,9 +35,6 @@ pub(crate) struct Test {
     /// Path to Cargo.toml
     #[clap(long, parse(from_os_str))]
     manifest_path: Option<PathBuf>,
-    /// Test all packages in the workspace
```

Comment:
(I note for @workingjubilee this is unrelated to this PR itself, it appears to be from 0.4.2 release.)